src/librustc_metadata/creader.rs

```diff
@@ -636,19 +651,22 @@ impl<'a> CrateReader<'a> {
             self.sess.span_fatal(item.span, &message);
         }
 
-        return ret
+        return ret;
     }
 
     /// Look for a plugin registrar. Returns library path, crate
     /// SVH and DefIndex of the registrar function.
-    pub fn find_plugin_registrar(&mut self, span: Span, name: &str)
+    pub fn find_plugin_registrar(&mut self,
+                                 span: Span,
+                                 name: &str)
                                  -> Option<(PathBuf, Svh, DefIndex)> {
-        let ekrate = self.read_extension_crate(span, &ExternCrateInfo {
-             name: name.to_string(),
-             ident: name.to_string(),
-             id: ast::DUMMY_NODE_ID,
-             should_link: false,
-        });
+        let ekrate = self.read_extension_crate(span,
+                                               &ExternCrateInfo {
+                                                   name: name.to_string(),
+                                                   ident: name.to_string(),
+                                                   id: ast::DUMMY_NODE_ID,
+                                                   should_link: false,
+                                               });
```

Comment:
IMO, the massive rightward drift here is ugly -- I think the original formatting looks much nicer.